### ISSUE TITLE
Distribution of call and ideas paper

### ISSUE THREAD

**Issue by kcopas** [gbif/doc-collections-idea-paper#1]
Planned contacts for consultation announcement and ideas paper—@akristensen can you help me with this in the next few days?

- [x] Workshop coordinators
- [x] CETAF: Karin Balilonoso <kbalilonoso@naturalsciences.be>, Laura Tilley (cc: Ana Casino)
- [x] DiSSCo: Dimitris, Wouter
- [x] iBOL: hjames@ibol.org[Hannah James] (cc: Donald)
- [x] CoL: Olaf Bánki
- [x] BHL: Grace Costantino, Martin Kalfatovic
- [ ] OBIS: Ward Appletans, Abby Benson, Sky Bristol
- [x] World Flora Online: William Ulate
- [x] ICZN: 
- [x] EMBL: Corrine Martin
- [ ] GenBank:
- [x] GEO BON: mailto:christian.langer@idiv.de[Christian Langer]
- [x] ABCD Network (via CAS): Maofang
- [x] ICIMOD: 
- [x] iNaturalist: post to [iNat forum](https://forum.inaturalist.org/t/upcoming-gbif-led-workshop-advancing-the-catalogue-of-the-worlds-natural-history-collection/10522)
- [x] WFCC: [Ma Juncai](mailto:ma@im.ac.cn), [Wu Linhuan](mailto:wulh@sun.im.ac.cn)
- [x] BGCI: [Paul Smith](mailto:paul.smith@bgci.org) , [Susanne Sharrock](mailto:suzanne.sharrock@bgci.org)
- [x] Species 360: Dalia Conde
- [x] BID community list
- [x] alliance steering committee
- [ ] other alliance signatory organizations

**Comment by akristensen** [comment]
Sure, so first finding the email info and add it to this list?

**Comment by kcopas** [comment]
Let's chat and maybe divvy up. 

I'll send at least the first one so you have a model (should be super brief and to the point).

**Comment by akristensen** [comment]
OK, cool. FYI got chicken Pox at home again, so will not be in tomorrow, but back again Thursday! I am in today! working on Joe's PPT for the next hour.

**Comment by akristensen** [comment]
> Planned contacts for consultation announcement and ideas paper—@akristensen can you help me with this in the next few days?
> 
> * [x]  Workshop coordinators
> * [x]  CETAF: Karin Balilonoso [kbalilonoso@naturalsciences.be](mailto:kbalilonoso@naturalsciences.be), Laura Tilley (cc: Ana Casino) Karsten (karsten.goedderz@cetaf.org ) twitter and added to dissco website: https://www.dissco.eu/advancing-the-catalogue-of-the-worlds-natural-history-collections/
> * [x]  DiSSCo: Dimitris, Wouter
> * [x]  iBOL: [hjames@ibol.org](mailto:hjames@ibol.org)[Hannah James] (cc: Donald) ANKR Add article to their Barcoed Bulletin?
> * [x]  CoL: Olaf Bánki
> * [x]  BHL: Grace Costantino, Martin Kalfatovic ANKR KalfatovicM@si.edu; CostantinoG@si.edu, Grace share via blog post and Social media
> * [ ]  OBIS: Ward Appletans, Abby Benson, Sky Bristol
> * [x]  World Flora Online: William Ulate
> * [x]  ICZN: ANKR iczn@nus.edu.sg (Gwynne LIM) Shared with the museum and on twitter.
> * [x]  EMBL: Corrine Martin
> * [ ]  GenBank: ANKR, there is no generic email, I am unsure who to contact: https://www.ncbi.nlm.nih.gov/research/postdocs/
> * [x]  GEO BON: [christian.langer@idiv.de](mailto:christian.langer@idiv.de)[Christian Langer] (confirmed incl. GEO BON newsletter on 18th of March.)ANKR will be in GEOBon Newsletter
> * [x]  ABCD Network (via CAS): Maofang
> * [x]  ICIMOD: Bandana Shakya Bandana.Shakya@icimod.org
> * [x]  iNaturalist: Carrie Seltzer
> * [x]  WFCC: ANKR ma@im.ac.cn; wulh@sun.im.ac.cn
> * [x]  BGCI: ANKR info@bgci.org paul.smith@bgci.org; suzanne.sharrock@bgci.org
> * [x]  Species 360: Dalia Conde
> * [x]  alliance steering committee ANKR (金恩植 Eun-Shik Kim <kimeuns@kookmin.ac.kr>; laetitia.navarro@idiv.de; Isayvani er <isayvani@gmail.com>; Robert Guralnick <robgur@gmail.com>; edwin.vanhuis@naturalis.nl; Donald Hobern <dhobern@gmail.com>; Kyle Copas <kcopas@gbif.org>; bbaptiste@universidadean.edu.com)
> * [ ]  other alliance signatory organizations

**Comment by kcopas** [comment]
Just hadn't saved my edits above—contacts now listed for WFCC and BGCI.

**Comment by akristensen** [comment]
> Just hadn't saved my edits above—contacts now listed for WFCC and BGCI.

Thanks,
Do you want to be in CC on the emails I send?

**Comment by kcopas** [comment]
Yes, please. And try to remember to change or generic-ize the first sentence's reference to the network (I know, I do it, too) :-)

**Comment by akristensen** [comment]
yeah so sorry, only realised it after I sent out to IBOL! feeling really embarrassed here! obviously pox on my brains @kcopas

**Comment by akristensen** [comment]
ICIMOD could it be https://www.icimod.org/team/laurie-ann-vasily?? @kcopas

**Comment by kcopas** [comment]
ask Maheva—I think the person she's working with on the Asian regional meeting is the prime contact for this…

**Comment by akristensen** [comment]
@kcopas 
GenBank: ANKR, there is no generic email, I am unsure who to contact: https://www.ncbi.nlm.nih.gov/research/postdocs/

**Comment by kcopas** [comment]
Let’s ask @dhobern if he’s got any contacts at GenBank…

**Comment by dhobern** [comment]
Yes - I can deal there. I'll do it today. Many thanks, Donald
-- 

*Donald Hobern*
+61 420511471

ORCID: 0000-0001-6492-4016 <https://orcid.org/0000-0001-6492-4016>
